pkg/universe.dagger.io/x/wangwei27494731@gmail.com/kustomize/kustomize.cue

```diff
@@ -0,0 +1,56 @@
+package kustomize
+
+import (
+	"universe.dagger.io/bash"
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+)
+
+// Apply a Kubernetes Kustomize folder
+#Kustomize: {
+	// Kubernetes source
+	source: dagger.#FS
+
+	// Optional Kustomization file
+	kustomization: string
```

Comment:
Nit: it would be cool if we could also provide a directory that contains kustomization.

You can leave it for a further PR but here's an example of disjunction : https://docs.dagger.io/1226/coding-style#favor-disjunctions-over-if-conditions